content/articles/getting-started-with-css/index.md

```diff
@@ -0,0 +1,194 @@
+```html
+<h2 style=" color: red;">Properties of CSS </h2>
+```
+
+## 2.Doing Various Manipulations to the Elements Selected
+### Using Names of Colors
+With CSS you can change the text. To do this you need to target the element who's text you need to modify with the appropriate selector. You then use the `color`attribute to specify the color you desire for your text as shown below. This can either be done in a separete style sheet or in your HTML file by adding the `style`element like I have in the example.
+```html
+<style>
+ h2{
+     color: red;
+ }
+ </style>
+ ```
+ This will change all the h2 elements to the color red.
+ 
+ Although not advisable in practise,you can also use inline styles to achieve the same results as illustrated in the example below
+ ```html
+ <h2 style="color: red;">Properties of CSS</h2>
+ ```
+ As pointed out above, a class is declared by adding a fullstop(.) accompanied by the class name in a style block.Then followed by a declaration block({}).Using the name of a color to style the class declaration will be like this;
+ ```html
+ <style>
+   .red-text{
+       color: red;
+   }
+ </style>
+ ```
+ In this example,.red-text is the class declaration.
+
+Introducing an Id attribute of Properties-of-CSS and giving it  a background color of black will be as shown in the example below.
+```html
+<style>
+ #Properties-of-CSS{
+    background-color: black;
+ }
+</style>
+```
+### Using Hex Codes
+Hex is the short name for Hexadecimal code. Hex code uses six hexadecimal numbers to specify colors in CSS. Some Hex values of different colors according to [Simmons](http://web.simmons.edu/) are;
+- black  #000000
+
+- maroon  #800000
+
+- yellow  #FFFF00
+
+- purple  #800080
+
+Representing color in Hex code will be like;
+```html
+<style>
+   p{
+   color: #800080;
+   }
+</style>
+```
+ But since it is hard to remember hex code,one can shorten it. This are examples from [freshersnow](https://tutorials.freshersnow.com/css/css-colors/);
+ - black  #000
+
+ - red  #F00
+
+ - cyan  #0FF
+
+### Using RGB colors
+RGB stands for red, green and blue. It is another way of representing colors in CSS.Each value could have a value from between o to 100% or 0 to 255. A value of 0 means none of the colors was used while a value of 255 or 100% means all of that color was used. The color black would result from all values (red, green and blue)being 0 while the color white would result from all the values being a 100%. Some of RGB values according to [rapidtables](https://www.rapidtables.com/web/color/RGB_Color.html) are as follows;
+- black  rgb(0, 0, 0)
+
+- red  rgb(255, 0, 0)
+
+- grey  rgb(128, 128, 128)
+
+RGB values can be styled in CSS as follows;
+```html
+<style>
+   p{
+   color: rgb(128, 128, 128);
+   }
+</style>
+```
+Let us use RGB values to represent an Id attribute;
+```html
+<style>
+   #Properties-of-CSS{
+                color: rgb(128, 128, 128);
+                }
+</style>
+```
+
+
+
```

Comment:
This is repetitive as you have already shown how to select elements in different ways exhaustively above. 
```suggestion
```